babel.config.js

```diff
@@ -1,20 +1,18 @@
 module.exports = {
     presets: [
-        '@babel/preset-env',
         '@babel/preset-typescript',
-        '@babel/preset-react',
-    ],
-    plugins: [
-        // this is needed to support `const enum`s - see https://babeljs.io/docs/en/babel-plugin-transform-typescript#caveats
-        'const-enum',
```

Comment:
This isn't related to emotion. I think this is just being removed because it's unused?